package_manifest.yaml

```diff
@@ -398,6 +398,7 @@ foreman_client_packages:
       - el6-puppet-5
       - el6-subscription-manager
```

Comment:
This doesn't sound correct if I look at the repos here

Reply:
What do you mean?

Reply:
In the client repos we don't install the subscription manager copr for EL6 so why is it needed for this repoclosure?